modules/services/pulseeffects.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pulseeffects;
+
+  presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
+
+in {
+  meta.maintainers = [ maintainers.jonringer ];
+
+  options.services.pulseeffects = {
+    enable =
+      mkEnableOption "Pulseeffects daemon. Used for advanced audio mixing.";
+
+    preset = mkOption {
+      type = types.lines;
+      default = "";
+      defaultText = literalExample "";
```

Comment:
You can remove this line, `defaultText` is only necessary if the regular `default` attribute cannot be represented in the manual.

Reply:
removed